packages/editor/editor.css

```diff
@@ -21,36 +21,31 @@
 	--radius-2: 6px;
 	--radius-3: 9px;
 	--radius-4: 11px;
-	/* Z Index */
-	--layer-hidden: -999999;
-	--layer-behind: -1;
-	--layer-culled-shape: 0;
-	--layer-text-label: 10;
-	--layer-collaborator-scribble: 10;
-	--layer-collaborator-brush: 20;
-	--layer-collaborator-shape-indicator: 30;
-	--layer-user-scribble: 40;
-	--layer-user-brush: 50;
-	--layer-user-indicator-selected: 60;
-	--layer-user-indicator-hovered: 70;
-	--layer-user-handles: 80;
-	--layer-user-snapline: 90;
-	--layer-background: 100;
-	--layer-user-indicator-hint: 110;
-	--layer-collaborator-cursor-hint: 120;
-	--layer-collaborator-cursor: 130;
-	--layer-grid: 150;
-	--layer-culled-shapes: 175;
-	--layer-canvas: 200;
-	--layer-shapes: 300;
-	--layer-overlays: 400;
-	--layer-in-front-overlay: 500;
-	--layer-in-front: 600;
-	--layer-bookmark-link: 999;
-	--layer-following-indicator: 1000;
-	--layer-blocker: 10000;
-
-	/* z index for text editors */
+
+	/* Canvas z-index */
+	--layer-cvs-hidden: -999999;
+	--layer-cvs-background: 100;
+	--layer-cvs-grid: 150;
+	--layer-cvs-shapes: 300;
+	--layer-cvs-overlays: 400;
+	--layer-cvs-blocker: 10000;
```

Comment:
children of the canvas element (the canvas and the infrontofthecanvas element no longer have z indices and are ordered dommy)